### PR TITLE
Add instructions on how to forward reverse lookups

### PR DESCRIPTION
**Pull request by cam8001**
*Description of changes:*

Adding a note about how to add forward rules for reverse lookups. This is particularly applicable when reverse lookups for AD need to be forwarded.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by joshbean** [comment]
Hello @cam8001 –

First, I'm sorry for the _terrible_ delay in responding to this PR! We strive to respond _much_ more quickly that what you're seeing here, and we will do so in the future.

Regarding the PR contents, I think this is a helpful addition to the documentation. Thank you! I'll provide a couple of review comments that I'd like to see addressed before we can merge this PR. However, I'm happy to just make those changes myself, on top of your commit, and then merge your PR, if that's easier for you. Just let me know!

File: doc_source/resolver-rules-managing.md [hunk 1]
```diff
@@ -179,4 +179,18 @@ Note the following:
...
+If you need to control reverse lookups in your VPC, you can add rules to your outbound resolver endpoint.
+
+### To create the reverse lookup rule
+
+ + Follow the procedure under [Creating Forwarding Rules](#resolver-rules-managing-creating-rules), up to step 5.
+ + When you specify your rule, enter the PTR record for the IP address you want a reverse lookup forwarding rule for.
+    + For example, if you need to forward lookups for addresses in the 10.0.0.0/23 range, enter a rules:
+      + 0.0.10.in-addr.apra
+      + 1.0.10.in-addr.apra
+  + Specify the server to forward these lookups to
```
Comment (joshbean) [inline doc_source/resolver-rules-managing.md:195]: Missing a period at the end of this sentence.

**Review by joshbean — CHANGES_REQUESTED** [review]
Except for the missing period, this looks good! But I'd move the whole thing up to the **Create forwarding rules** section. As noted in this procedure, it's really just an extension of that one, so I'd put it up there closer in context, rather than at the end of all the other sections.

File: doc_source/resolver-rules-managing.md (hunk 1 above)
Comment (cam8001) [reply doc_source/resolver-rules-managing.md:195]: Hey thanks! I’d completely forgotten about this. I notice before the example at the end I’ve put ‘enter a rules’. ‘add rules’ or ‘enter rules’ would probably be better. I’m phoneposting atm but can take a look later.

Comment (joshbean) [reply doc_source/resolver-rules-managing.md:195]: Were you able to take another look at this? Again, I'm happy to just make the changes myself by building on top of your commit, if that's easier for you.

Comment (cam8001) [reply doc_source/resolver-rules-managing.md]: OK, just pushed some updates. FWIW, there seems to be bunch of trailing spaces on the headings in this file, which VS Code keeps complaining about.

**Review by joshbean — APPROVED** [review]

File: doc_source/resolver-rules-managing.md (hunk 1 above)
Comment (joshbean) [reply doc_source/resolver-rules-managing.md]: Yeah, sorry about the trailing spaces. Our authoring and publishing system sometimes adds those.

This looks good! I'll go ahead and merge the PR. I still want to move these steps up closer to the steps for creating forwarding rules, but I'll do that after the merge.

Thanks for your PR!

**Pull request merged**